examples/scripts/deconv_datagen_foam.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Generation of Blurred Data for NN Training
+==========================================
```

Comment:
Better not to have the same title as other example scripts.

Reply:
I changed it in the other script (`deconv_datagen_bsds.py`) to "Generation of Blurred Data (Natural Images) for NN Training", but it is a long title.